test/exprtest/blas1_axpy_copy_test.cpp

```diff
@@ -104,8 +99,7 @@ void run_test(const combination_t<scalar_t> combi) {
   ex.get_policy_handler().wait(axpy_event);
 
   // Copy the result back to host memory
-  auto getResultEv =
-      ex.get_policy_handler().copy_to_host(gpu_y_v, v_y.data(), y_dim);
+  auto getResultEv = utils::quantized_copy_to_host<scalar_t>(ex, gpu_y_v, v_y);
```

Comment:
Since we have quantized copy to host, do we have a quantized copy to a device? Explicit data movement is crucial for getting performance since it separates the computation from the data movement. Just to mention that you don't need to add it here, since it is a test and it does not matter whether or not you explicitly copy the data.

Reply:
@mehdi-goli I think the quantize to device happens as part of the call to ```utils::make_quantized_buffer```

Reply:
So you mean it quantizes and copies it to the device?

Reply:
Yeah you can see it [here I believe ](https://github.com/ProGTX/sycl-blas/commit/cf48142082bea1ea864fc438ed13114cb9e69d75#diff-64279e7f75aee013cccd104e34670f11R84) where this is called from within ```utils::make_quantized_buffer```.

Reply:
Correct, these are the steps `make_quantized_buffer` takes when having to quantize:
 * 1. Constructs a buffer to hold the input data, which is float or double
 * 2. Copies input data to the buffer
 * 3. Constructs a buffer to hold data of scalar_t
 * 4. Performs quantization from input data to scalar_t

For `float` and `double` it can skip the quantization steps.

Reply:
Per the other discussion as this is just a helper function it seems fine, if a user wants more explicit control over when exactly the data is quantized/copied to the device they can just do it the regular way without the helper.